[Feature #114298229] Show notification on meetup creation

andela-aonyango opened the pull request:
Added a notification to indicate that the meetup creation was successful.

Like Dribbble, the meetup controller redirects the user to the FAQ page on success,
and there's a notification letting them know that their actions were successful and that
they will be contacted on approval.

Refactored the mail-sending portion of the meetup controller to make
it clearer, as well as to separate the addressees clearly into the user
creating the email and the admins who are CCed on every meetup creation.

andela-aonyango committed:
[Feature #114298229] Show notification on meetup creation

Added a notification to indicate that the meetup creation was successful. Like
Dribbble, the meetup controller redirects the user to the FAQ page on success
and there's a notification letting them know that their actions were successful
and that they will be contacted on approval.

andela-aonyango committed:
[Feature #114298229] Show notification on meetup creation (Edit)

Modified the changes from commit 6eade00 to move the alert to a
partial template and include that template in the FAQ page where
the alert is displayed.

Also modified the meetup CSS file to center the alert text and close
the top margin.

unicodeveloper commented:
When I try to create a new meetup, I get this error: @andela-aonyango 

<img width="1113" alt="screen shot 2016-02-25 at 6 02 29 pm" src="https://cloud.githubusercontent.com/assets/2946769/13327413/0bce04e2-dbea-11e5-98ee-894b158fa128.png">

unicodeveloper commented on app/Http/Controllers/MeetupController.php:
```diff
@@ -71,6 +73,6 @@ public function sendEmailOnMeetupCreation(Meetup $meetup)
         Mail::send('emails.meetup-created', compact('user', 'meetupDetails'), function ($message) use ($user, $recipients) {
```
**Code Quality - Critical**

You are storing the admin emails in `$recipients` variable which is an array. You are also storing the user's email in `$recipients[]` variable which is also an array. Should that be two different named variables where the first is an array and the second is a variable that just holds a string?

andela-aonyango replied:
As it is, I think it's okay because the `$recipients` variable holds the emails of all the intended recipients of a meetup notification email: the admins and the user who created the meetup. The user's email is just appended to the end of the array after the admins' emails.

However, I didn't know of the `cc()` method at the time. I'll separate the addressees into primary (the user => to) and secondary (admins => cc).

But that's not the cause of the error. That's because the ADMIN_EMAILS variable (which was sent earlier today on Slack) is not set on Heroku for the PR build, but it's set for the `staging` build.

Do I have access to the PR build settings as well?

unicodeveloper replied:
It would have been easier and easily understandable if you used `array_push`.

..but every environment variable works for staging and all PR builds.

andela-aonyango committed:
[Feature #114298229] Show notification on meetup creation (Refactor)

Refactored the mail-sending portion of the meetup controller to make
it clearer, as well as to separate the addressees clearly into the user
creating the email and the admins who are CCed on every meetup creation.

andela-aonyango committed:
Merge branch 'staging' into feature/show-notification-meetup-creation

Conflicts:
	app/Http/Controllers/MeetupController.php

unicodeveloper commented:
This is still not working @andela-aonyango . Why?

<img width="1103" alt="screen shot 2016-02-29 at 2 56 33 pm" src="https://cloud.githubusercontent.com/assets/2946769/13396419/bb82750a-def4-11e5-88d9-e48c5adaf8d0.png">

andela-aonyango commented:
@unicodeveloper, the issue, as I mentioned in a previous exchange, was that the environment variables (ADMIN_EMAILS) were not (automatically) set in the PR build. I've manually set them now but I don't know if the other PR builds will pick them up. The staging build has no issue.

So, it will work in this build but I don't know how subsequent builds will work.
